app/ide-desktop/lib/client/package.json

```diff
@@ -36,6 +37,7 @@
     "electron-builder": "^22.14.13",
     "electron-notarize": "1.2.2",
     "enso-copy-plugin": "^1.0.0",
+    "enso-studio-common": "^1.0.0",
```

Comment:
We do not have "Enso Studio" anymore. It is super old name and instead of that we are using "Enso" now. In case "Enso" would not be sufficient, we can use "Enso IDE", but "Enso" would be better and more consistent.